src/applications/facility-locator/components/SearchResultsHeader.jsx

```diff
@@ -10,12 +10,12 @@ import { LocationType } from '../constants';
 import { connect } from 'react-redux';
 
 export const SearchResultsHeader = ({
+  specialties,
   results,
   facilityType,
   serviceType,
   context,
   inProgress,
-  specialtyMap,
```

Comment:
If `specialties` is still an object, I kind of like the name `specialtyMap`, since it is more self-documenting.

What was your reason for changing the prop name?

Reply:
that's fine I can leave specialtyMap